process/wg-lifecycle-documents/ai_ml_incubating_stage.md

```diff
@@ -0,0 +1,60 @@
+## AI/ML Working Group incubation application
+
+### List WG Chairs
+
+The WG must have a minimum of 1 Chair
+
+* Mihai Maruseac, Google, mihaimaruseac
+* Jautau "Jay" White, Microsoft, camaleon2016
+
+### Working Group (WG) has met all Sandbox requirement
+
+* Applying directly to Incubating
```

Comment:
Add how the Sandbox requirements are completed:
Proposal of scope for review by TAC
This is to help ensure limited overlap with existing WGs
Have at least 3 interested individuals from at least 2 different organizations supporting the proposal (https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md)
1 TAC sponsor - Was it originally Dan A? Is it now Jay W?
TAC sponsor agrees to attend WG meetings regularly
TAC sponsor does not need to have a formal role in WG, e.g., chair
TAC sponsor requests TAC approval

Reply:
Thank you! Included these ones and then I'll work on the docs on the WG side.